HACKING.rst.txt

```diff
@@ -138,6 +138,11 @@ will object to having it fixed.  It may be necessary to add some warning or
 deprecation message for the changed behavior.  This should be part of the
 review process.
 
+*Note: please avoid making pull requests that only change code style, e.g.
+fixing some PEP8 issues in a file. On average, things like PEP8 cleanups are
```

Comment:
Instead of the 

> On average, things like PEP8...

 I would use 

> Generally speaking, PRs that exclusively focus on PEP8 style and similar stylistic changes are discouraged. They are often not worth cluttering the Git commit history. Instead they are best placed inside functional change PRs either by reminding other PR owners during a code review or combined with a more substantial change.

Wording is not super smooth but something along the way makes the message a bit more clearer.

Reply:
Good suggestion. I've adapted your wording a bit.